src/ray/common/ray_config_def.h

```diff
@@ -518,6 +518,10 @@ RAY_CONFIG(int64_t, log_rotation_backup_count, 5)
 /// as failed.
 RAY_CONFIG(int64_t, timeout_ms_task_wait_for_death_info, 1000)
 
+/// The core worker heartbeat interval. During heartbeat, it'll
+/// report the loads to raylet.
+RAY_CONFIG(int64_t, core_worker_internal_heartbeat_ms, 1000);
```

Comment:
for clarification, we only use this for testing right?

Reply:
I don't think so, it's controlling the heartbeat, like how frequent the core worker sent the loads info to raylet.

For extreme big machine, with a lot of workers locally, the user should give it a bigger value to avoid overload raylet.

Right now, we also have other logics there, for which I feel sad about :(, and we should clean it up later.